routers/admin_router.js

```diff
@@ -40,11 +41,15 @@ router.use(isAuth, isAdmin);
 
 router
     .patch('/make-admin', async (req, res) => {
+        const { error } = adminValidation(req.body);
+        if (error) {
+            return res.status(400).json({ message: error.details[0].message });
+        }
         const user = await users.findOne({ _id: req.body.id });
         if (user) {
             user.isAdmin = true;
             await user.save();
-            return res.status(200).json({ message: 'Администратор успешно назначен' });
+            res.status(200).json({ message: 'Администратор успешно назначен' });
```

Comment:
Sure, that return isn't required here? If your code calls .json method twice in route - there will be an error.